server/server/views.py

```diff
@@ -0,0 +1,5 @@
+from pyramid.view import view_config, view_defaults
```

Comment:
I'm not sure this file is still needed. Anyway you already have package with name `server.views`, and you are adding module with same name. There might be some conflicts on importing.